include/ascend/dcmi_interface_api.h

```diff
@@ -0,0 +1,1627 @@
+﻿/*
```

Comment:
Do you know if we are allowed to redistribute this file?
What is its license?

Reply:
That header file I committed is wrong, please replace it with this: [https://github.com/Ascend/ascend-npu-exporter/blob/master/devmanager/dcmi/dcmi_interface_api.h](https://github.com/Ascend/ascend-npu-exporter/blob/master/devmanager/dcmi/dcmi_interface_api.h) . This is Apache 2.0 License and can safely redistribute, I have tested this new header and it is okay for compiling and running.

Reply:
I'm sorry, I need to confirm this.

Reply:
Okay, the new opensourced header file needs to do minor changes, I've updated and committed the modification.